catroid/src/main/res/values-in/strings.xml

```diff
@@ -1711,7 +1711,7 @@ needs read and write access to it. You can always change permissions through you
     <string name="formula_editor_logic_comparison">Operator perbandingan</string>
     <string name="formula_editor_object_general">Properti Umum</string>
     <string name="formula_editor_object_movement">Sifat gerak</string>
-    <string name="formula_editor_device_visual_detection">Visual sensors</string>
+    <string name="formula_editor_device_face_detection">Visual sensors</string>
```

Comment:
undo this changes, nothing to do here, it will eventually be overwritten by the translations